src/components/znkModuleResults/znkModuleResult.service.js

```diff
@@ -0,0 +1,92 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.znkModuleResults').service('ZnkModuleResultsService', [
+        'InfraConfigSrv', '$log', '$q', 'StorageSrv', 'UtilitySrv',
+        function (InfraConfigSrv, $log, $q, StorageSrv, UtilitySrv) {
+
+            var znkModuleResultsService = {};
+            var MODULE_RESULTS_GUIDS_PATH = StorageSrv.variables.appUserSpacePath + '/moduleResults';
+            var MODULE_RESULTS_PATH = 'moduleResults';
+
+            function _isValidNumber(number){
```

Comment:
look everywhere in infra. didn't found.
I moved this function to UtilitySrv and delete it from this service